lib/transaction/transaction.js

```diff
@@ -429,10 +431,13 @@ Transaction.prototype.fromObject = function fromObject(arg) {
     this._fee = transaction.fee;
   }
   this.nLockTime = transaction.nLockTime;
-  this.version = transaction.version;
+  this.version = transaction.version == null ? CURRENT_VERSION : transaction.version;
```

Comment:
Triple equal

Reply:
Actually, will `CURRENT_VERSION || transaction.version` suffice for this?

Reply:
@KamuelaFrancoI did this on a purpose. `x == null` will only work if `x` is null or undefined. `CURRENT_VERSION || transaction.version` will set the `version` field to the `CURRENT_VERSION` value, even if you use `0` as version, which isn't the desirable behavior